plugins/ruby-foreman-chef/debian/rules

```diff
@@ -11,15 +11,4 @@
 
 PLUGIN = foreman_chef
 
-build:
-	cp cache/* /usr/share/foreman/vendor/cache/
-	cp $(PLUGIN).rb /usr/share/foreman/bundler.d/
-	cd /usr/share/foreman && ( \
-		/usr/bin/foreman-ruby /usr/bin/bundle install --local && \
-		/usr/bin/foreman-ruby /usr/bin/bundle exec rake plugin:assets:precompile[$(PLUGIN)] RAILS_ENV=production DATABASE_URL=nulldb://nohost \
-	)
-	GEM_PATH=$$(cd /usr/share/foreman && /usr/bin/foreman-ruby /usr/bin/bundle show $(PLUGIN)) && \
-		cp -rp $${GEM_PATH}/public/assets/$(PLUGIN) ./
-
-%:
-	dh $@ 
+include /usr/share/foreman/foreman-plugin.mk
```

Comment:
This doesn't have `npm` assets but it does call `/usr/bin/npm install --no-audit --no-optional --unsafe-perm`. In the RPM macros we have parameters to determine what it does and doesn't do. Perhaps foreman-plugin.mk needs the same? Alternatively it could look for `package.json` and similar for apipie cache + asset pipeline

Reply:
guess why this PR is called "try" ;-)